### ISSUE TITLE
Lint reports on using color map-key as function parameter

### ISSUE THREAD

**Issue by bartveneman** [causes/scss-lint#344]
````scss
// _vars.scss
$colors: (
  blue: #0000ff,
  red: #ff0000
);
````
````scss
// _functions.scss
@function color($name) {
  @return map-get($colors, $name);
}
````
````scss
// main.scss
.myElement {
  color: color(red);
}
````
Scss-lint now reports an error on `main.scss` because `ColorKeyword: Color 'red' should be written in hexadecimal form as '#ff0000'`. I might be missing something, but I don't think this is desirable behaviour.

**Comment by lencioni** [comment]
Thanks for the report @bartveneman. However, I think this is a duplicate of #175.

**Comment by bartveneman** [comment]
You're absolutely right. Sorry about that.

**Comment by lencioni** [comment]
No worries! Thanks for the report!
